### ISSUE TITLE
./configure && make fails for OMR

### ISSUE THREAD

**Issue by mgaudet** [eclipse/omr#210]
Similarly, `./configure --enable-OMR_EXAMPLE && make` also fails: 

```
make[2]: Entering directory `/home/magaudet/open/omr/gc/base'
g++  -I. -I../include -I../stats -I../structs -Istandard -Isegregated  -DUT_DIRECT_TRACE_REGISTRATION -I../../include_core -I../../nls -DLINUX -D_REENTRANT -D_FILE_OFFSET_BITS=64 -DJ9HAMMER -c  -MMD -MP -fno-exceptions -fno-rtti -fno-threadsafe-statics -fPIC -ggdb -m64 -Wreturn-type -Werror -Wall -Wno-non-virtual-dtor -O3 -fno-strict-aliasing -funroll-loops  -o AddressOrderedListPopulator.o AddressOrderedListPopulator.cpp
In file included from AddressOrderedListPopulator.hpp:22:0,
                 from AddressOrderedListPopulator.cpp:19:
ObjectHeapBufferedIteratorPopulator.hpp:23:31: fatal error: objectdescription.h: No such file or directory
 #include "objectdescription.h"
                               ^
compilation terminated.
make[2]: *** [AddressOrderedListPopulator.o] Error 1
```

**Comment by youngar** [comment]
Hi, I can look into this if no one else is already.  @mgaudet, have you started examining this?

**Comment by mgaudet** [comment]
Nope -- haven't looked into it at all :)

**Comment by youngar** [comment]
We have a similar Issue #67.

The failure in the output is because OMRGLUE is not set correctly, and processor is unable to find the OMR glue header files while building.  I think we can make this more apparent by checking the directory OMRGLUE for a glue header file, and printing a helpful error message.

In README.md, it specifies
> The minimal invocation of configure is:
> $ ./configure OMRGLUE=./example/glue

So neither `./configure && make` or `./configure --enable-OMR_EXAMPLE` are actually officially supported.  I do like the idea of supporting both of these options, though.  We can use a default OMRGLUE directory of `example/glue`, which will allow both invocations.

**Comment by youngar** [comment]
Closed via #227

**Comment by youngar** [comment]
The AC_CHECK_FILE() errors out when cross compiling. See pull request #270.
